builder/proxmox/step_upload_additional_isos.go

```diff
@@ -0,0 +1,68 @@
+package proxmox
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+
+	"github.com/Telmate/proxmox-api-go/proxmox"
+	"github.com/hashicorp/packer/helper/multistep"
+	"github.com/hashicorp/packer/packer"
+)
+
+// stepUploadAdditionalISOs uploads all additional ISO files that are mountet
+// to the VM
+type stepUploadAdditionalISOs struct{}
+
+type uploader interface {
+	Upload(node string, storage string, contentType string, filename string, file io.Reader) error
+}
+
+var _ uploader = &proxmox.Client{}
+
+func (s *stepUploadAdditionalISOs) Run(ctx context.Context, state multistep.StateBag) multistep.StepAction {
+	ui := state.Get("ui").(packer.Ui)
+	client := state.Get("proxmoxClient").(uploader)
+	c := state.Get("config").(*Config)
+
+	for idx := range c.AdditionalISOFiles {
+		if !c.AdditionalISOFiles[idx].shouldUploadISO {
+			state.Put("additional_iso_files", c.AdditionalISOFiles)
+			return multistep.ActionContinue
```

Comment:
This should be a `continue`, right? Otherwise we'll skip uploading any iso:s later in the list

Reply:
Yeah you're right. Thanks for the heads up :+1: